code/src/test/java/com/expediagroup/sdk/core/http/HeadersJavaTest.java

```diff
@@ -0,0 +1,51 @@
+package com.expediagroup.sdk.core.http;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+public class HeadersJavaTest {
+
+    @Test
+    @DisplayName("should access default builder static method")
+    public void default_static_builder_IsAccessible_from_Java() {
```

Comment:
Can we please stick to the camel case convention?

Reply:
Done